simphony/cuds/material.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+from simphony.core.data_container import DataContainer
+
+
+class Material(object):
+    """A class that represents a material.
+
+    Class describes a material and its data.
+
+    Attributes
+    ----------
+    description : str
+        textual description of the material
+    uid : uuid.UUID
+        the id of the material. If None, then it is generated
+    data : DataContainer
+        data associated with this material
+
+    """
+    def __init__(self, uid=None, data=None, description=""):
```

Comment:
> I prefer an explicit method such as get_material(uid). BTW, it is not clear for me why a Material class should have such a method at all, since it is the material itself, not a container. 

`get_material` is a method for `Materials` and not `Material`